data/sql/updates/pending_db_world/rev_1582975268828510900.sql

```diff
@@ -0,0 +1,752 @@
+(9783,'deDE','Es tut weh, die Lebewesen des Todesmoors umbringen zu müssen, aber wir können sie nicht retten. Das einzige, was wir tun können, ist zu versuchen, das Gebiet wiederherzustellen und eine neue Generation zu ernähren.',18019),
+(9785,'deDE','Habt Ihr schon mit den Uralten gesprochen, $N?',18019),
+(9786,'deDE','Habt Ihr die Ruinen von Boha\'mu schon untersucht? Könnt Ihr euch vorstellen, wie sie zu ihren Glanzzeiten ausgesehen haben?',18019),
+(9789,'deDE','Die Grollhufe in Nagrand sind eine gute Herausforderung. Ich habe selbst eine ganze Reihe von ihnen erlegt.$B$BWie läuft es bei Euch?',18019),
+(9791,'deDE','Hattet Ihr Erfolg bei der Jagd auf die Marschenfänge?',18019),
+(9804,'deDE','Benutzt die Nagrandkirschen bevor Ihr Euch so tief in den See stürzt.',18019),
+(9810,'deDE','Der Irdene Ring wird helfen, wo er nur kann.',18019),
+(9813,'deDE','Ja, $C?',18019),
+(9817,'deDE','Ihr seid hier, um das Kopfgeld für Fyashe einzufordern? Das ist die beste Nachricht, die ich heute gehört habe.',18019),
+(9819,'deDE','Schlafen sie?',18019),
+(9824,'deDE','Die Aufzeichnungen, $N. Habt Ihr sie beendet?',18019),
+(9825,'deDE','Habt Ihr die Essenzen, die ich benötige, besorgt, $C?',18019),
+(9826,'deDE','Ihr wisst vom Violetten Auge? Ihr kommt mir nicht bekannt vor.',18019),
+(9831,'deDE','Habt Ihr das Fragment besorgt, $C? Ihr solltet dabei besser keine Zeit verschwenden.',18019),
+(9832,'deDE','Wie kommt Ihr mit der Suche nach den Schlüsselfragmenten voran, $N?',18019),
+(9833,'deDE','Kommt Ihr mit der Verbesserung der Sicherheit unserer Straßen voran?',18019),
+(9835,'deDE','Wie läuft der Kampf gegen die Ango\'rosh?',18019),
+(9836,'deDE','Wer seid Ihr und was wollt Ihr?',18019),
+(9839,'deDE','Seid Ihr fertig? Habt Ihr Oberanführer Schmetterfaust besiegt?',18019),
+(9854,'deDE','Habt Ihr schon den Frieden gefunden, der einen überkommt, wenn man seinen Platz in der Natur gefunden hat? Seid Ihr Jäger oder Gejagter?',18019),
+(9855,'deDE','Habt Ihr inzwischen gelernt, Euren Herzschlag zu zügeln? Ruhig und geduldig wie ein stiller See zu sein und auf den richtigen Moment zu warten, um die Beute zu erlegen?',18019),
+(9857,'deDE','Ich bleibe hier einfach noch ein bisschen liegen. Es geht mir sicher bald besser.',18019),
+(9858,'deDE','Ich nehme nicht an...<hust>...dass Ihr Euch schon...<hust>..um die Dornzehrer gekümmert habt?',18019),
+(9862,'deDE','Der Luftgeist meint, dass die Finsterblut in Sonnenwind nach etwas suchen...',18019),
+(9865,'deDE','Für jeden meiner Brüder, den sie umgebracht haben, müssen zehn von ihnen sterben!',18019),
+(9878,'deDE','Die Finsterblut sind ein ruchloser Stamm der Zerschlagenen, $N. Sie mögen vielleicht aussehen wie Kurenai, aber tatsächlich sind sie wilde, mordende Tiere.',18019),
+(9894,'deDE','Habt Ihr Lord Klaq gefunden und vernichtet?',18019),
+(9895,'deDE','Habt Ihr Sumpfschläger gefunden?',18019),
+(9900,'deDE','Ihr seid zurück, $R. Ist Gava\'xi tot?',18019),
```

Comment:
better using `''` instead of `\'` to escape SQL

Reply:
Updated - it's the same in my other PRs because it's a HeidiSQL outpot.